### PR TITLE
Adds `contains` and `get` finalize commands

### PR DESCRIPTION
**Pull request by howardwu**
## Motivation

This PR adds `contains` and `get` finalize commands.

### CODE REVIEW

File: synthesizer/src/program/finalize/command/contains.rs [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    type CurrentNetwork = Testnet3;
+
+    #[test]
+    fn test_parse() {
+        let (string, get) = Contains::<CurrentNetwork>::parse("contains account[r0] into r1;").unwrap();
+        assert!(string.is_empty(), "Parser did not consume all of the string: '{string}'");
+        assert_eq!(get.mapping, Identifier::from_str("account").unwrap());
+        assert_eq!(get.operands().len(), 1, "The number of operands is incorrect");
+        assert_eq!(get.key, Operand::Register(Register::Locator(0)), "The first operand is incorrect");
+        assert_eq!(get.destination, Register::Locator(1), "The second operand is incorrect");
```
Comment (d0cd) [inline synthesizer/src/program/finalize/command/contains.rs]: Rename `get` to `contains`

**Comment by howardwu** [comment]
I'll sample parameters after this design looks good.

File: synthesizer/src/program/finalize/command/remove.rs [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+    type CurrentNetwork = Testnet3;
+
+    #[test]
+    fn test_parse() {
+        let (string, set) = Remove::<CurrentNetwork>::parse("remove account[r1];").unwrap();
+        assert!(string.is_empty(), "Parser did not consume all of the string: '{string}'");
+        assert_eq!(set.mapping, Identifier::from_str("account").unwrap());
+        assert_eq!(set.operands().len(), 1, "The number of operands is incorrect");
+        assert_eq!(set.key, Operand::Register(Register::Locator(1)), "The first operand is incorrect");
```
Comment (d0cd) [inline synthesizer/src/program/finalize/command/remove.rs]: Rename `set` to `remove`

**Review by d0cd — APPROVED** [review]
LGTM! Left some minor nits.

File: synthesizer/src/program/finalize/command/remove.rs (hunk 2 above)
Comment (howardwu) [reply synthesizer/src/program/finalize/command/remove.rs]: Fixed in [4b7bd52](https://github.com/AleoHQ/snarkVM/pull/1666/commits/4b7bd5221a1e6175710087658420b6e933c27814)

File: synthesizer/src/program/finalize/command/contains.rs (hunk 1 above)
Comment (howardwu) [reply synthesizer/src/program/finalize/command/contains.rs]: Fixed in [4b7bd52](https://github.com/AleoHQ/snarkVM/pull/1666/commits/4b7bd5221a1e6175710087658420b6e933c27814)

**Pull request merged**